tty/pc-tty/Makefile

```diff
@@ -3,9 +3,26 @@
 #
 # Copyright 2017, 2018, 2019 Phoenix Systems
 # Copyright 2001, 2005-2006 Pawel Pisarczyk
-#
 
-$(PREFIX_PROG)pc-tty: $(addprefix $(PREFIX_O)tty/pc-tty/, ttypc.o ttypc_vt.o ttypc_vtf.o ttypc_vga.o ttypc_kbd.o ttypc_bioskbd.o) $(PREFIX_A)libtty.a
-	$(LINK)
 
-all: $(PREFIX_PROG_STRIPPED)pc-tty
+NAME := pc-tty
+SRCS := $(wildcard tty/pc-tty/*.c)
+STATIC_LIBS := libtty
+
+include binary.mk
+
+# NOTE: multiple binaries can be created by redefining variables and including binary.mk again
+
+# below the manual way for expilict compile dependency
+
+
+#OBJS := $(addprefix $(PREFIX_O)tty/pc-tty/, ttypc.o ttypc_vt.o ttypc_vtf.o ttypc_vga.o ttypc_kbd.o ttypc_bioskbd.o)
+
+#$(OBJS) : libtty
+
+#$(PREFIX_PROG)pc-tty: $(OBJS) $(PREFIX_A)libtty.a
+#	$(LINK)
+
+#pc-tty: $(PREFIX_PROG_STRIPPED)pc-tty
+
```

Comment:
probably in this manual way you forgot to add
.PHONY: pc-tty

Reply:
yep, actually there are more things missing, but I wanted to provide minimal working example.

I'll try to rewrite all `phoenix-rtos-devices` and will see what I learn along the way.

By using template (and including all Makefiles) I would like to add other automatic targets, like:
- `$(NAME)-clean` -> removes binaries (or libs), objects and dependency files for single component
- `$(NAME)-install` -> installs binary to predefined rootfs location
- `$(NAME)-uninstall` :question: